fatrace.8

```diff
@@ -72,6 +75,73 @@ prints the devices' major and minor number and the inode number. To examine
 such a process in more detail, you should consider using
 .BR strace (1).
 
+.RE
+
+In JSONL format, the line is a JSON-encoded object with the following fields:
```

Comment:
`--json` is a very complex feature that definitively requires updating at least some of the tests to cover the json output. Perhaps copy `tests/fatrace-{comm,currentmount,user` to a new `*-json` and adjust the expected output accordingly? I recommend to also pipe the output to `jq` to make sure the syntax is ok.

Reply:
Pushed to #51.